### ISSUE TITLE
polishing strategy and rounds

### ISSUE THREAD

**Issue by binlu1981** [Nextomics/NextPolish#43]
Hi,

I have an assembly generated using wtdbg2 which could use wtpos-cns performing polishment. Will I get a better result through performing additional long- or short-reads polishing steps using the NextPolish? Would it produces conflict results using different polishing tools?
How many rounds can reach best status and how to determine?

**Comment by moold** [comment]
Yes, see our paper, `Hu, Jiang, et al. "NextPolish: a fast and efficient genome polishing tool for long read assembly." Bioinformatics (Oxford, England) (2019)`

**Comment by binlu1981** [comment]
Thanks for quickly reply. Your paper mainly focus on short reads corrections and documented that two rounds is good enough, so I plan to perform one round PacBio long-reads + two round short-reads correction using wtpos-cns, and finally use NextPolish to do two round short-reads correction. Will it work?
The assembly is ~2Gb and short-reads are about 100 Gb. My severs has ~330 Gb memory, is it enough to run NextPolish?

**Comment by moold** [comment]
The memory is enough, NextPolish does not require much memory. I do not test wtpos-cns, for NextPolish, two round short-reads correction is enough, but if you want a more accurate assembly, you can try more iterations.
